
class ClienteRepetido(Exception):
    def __init__(self):
        return ("El cliente ya existe") #no pongo return porque el def init no me devuelve nada 
    
        
       
